ui/jobs/components/drk.ts

```diff
@@ -1,12 +1,14 @@
+import { Bars } from '../bar';
 import { kAbility } from '../constants';
 import { computeBackgroundColorFrom } from '../utils';
+import { JobDetail } from '../../../types/event';
 
-let resetFunc = null;
-let tid1;
-let tid2;
-let tid3;
+let resetFunc: ((bars: Bars) => void) = (_bars: Bars) => undefined;
+let tid1: ReturnType<typeof setTimeout>;
```

Comment:
I still am confused about this.  Are you planning to run jobs inside of node?  I think this is browser only, and really is `number`?

Reply:
I think whatever it's real type is, it's always`ReturnType<typeof setTimeout>`

Reply:
:+1: